src/docs/upgrading.dox

```diff
@@ -1,5 +1,40 @@
 /*! @page upgrading Upgrading WiredTiger applications
 
+@section version_1130 Upgrading to Version 11.3.0
+<dl>
+
+<dt>Per-file configuration "huffman_key"</dt>
```

Comment:
Shouldn't this be `huffman_value` since that's the thing that was changed?

Reply:
Good catch! Addressed in [f8282ee](https://github.com/wiredtiger/wiredtiger/pull/11194/commits/f8282eeca6963f774a55f939a011acaf71258e43)